iceoryx_binding_c/test/moduletests/test_runtime.cpp

```diff
@@ -85,8 +83,8 @@ TEST_F(BindingC_Runtime_test, GetInstanceNameIsNullptr)
 
 TEST_F(BindingC_Runtime_test, GetInstanceNameLengthIsLessThanAppNameLength)
 {
-    constexpr char ACTUAL_APP_NAME[iox::MAX_PROCESS_NAME_LENGTH + 1] = "/chucky";
-    constexpr char EXPECTED_APP_NAME[iox::MAX_PROCESS_NAME_LENGTH + 1] = "/chuck";
+    constexpr char ACTUAL_APP_NAME[iox::MAX_PROCESS_NAME_LENGTH + 1] = "chucky";
+    constexpr char EXPECTED_APP_NAME[iox::MAX_PROCESS_NAME_LENGTH + 1] = "chucky";
```

Comment:
You should also update ``APP_NAME_BUFFER_LENGTH``. Otherwise the test doesn't make sense anymore.